docs/reconciliation.md

```diff
@@ -73,6 +73,36 @@ annotations:
   eno.azure.io/disable-updates: "true"
 ```
 
+### Overrides
```

Comment:
By far the most common use case for this will be supporting external controller ownership of a certain subset of fields. Might be worth exploring that here?